src/style.css

```diff
@@ -1,3 +1,89 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid using the '*' selector as it impacts performance. Instead, set styles only for elements that require them.